internal/config/flavor.go

```diff
@@ -13,14 +13,12 @@ var hoverYaml string
 
 // GetHoverFlavorYaml returns the Hover yaml file
 func GetHoverFlavorYaml() string {
+	if len(hoverYaml) == 0 {
+		hoverYaml = "hover.yaml"
+	}
 	return hoverYaml
 }
 
-// SetDefaultFlavorFile sets the default hover.yaml
-func SetDefaultHoverYamlFile() {
-	hoverYaml = "hover.yaml"
-}
-
```

Comment:
These changes were needed, because the config needed to be loaded for `init-packaging`. It removes the need to call `SetDefaultHoverYamlFile()`.

Reply:
Makes, sense. I was not aware that `len` returns 0 on a nil

Reply:
Strings are never nil they are just empty